src/commands/scp.ts

```diff
@@ -84,9 +84,9 @@ const scpAction = async (args: yargs.ArgumentsCamelCase<ScpCommandArgs>) => {
   if (!result) {
     throw "Server did not return a request id. Please contact support@p0.dev for assistance.";
   }
-  const { request, privateKey } = result;
+  const { request, privateKey, linuxUserName } = result;
 
-  const data = requestToSsh(request);
+  const data = requestToSsh(request, { gcloud: { linuxUserName } });
```

Comment:
separation of concerns here, as well

Reply:
This object indicates that this is gcloud-specific data.
This `scp` and the `shared` module where `requestToSsh` lives are shared between ssh and scp, not between CSPs.
You had another comment in `requestToSsh` to call out to CSP-specific modules which I will do.
I'm not sure what else to do on this line.

Reply:
See https://github.com/p0-security/p0cli/pull/90#discussion_r1697685917